Specs/DateTime/Spanish/DateTimeModel.json

```diff
@@ -6714,52 +6702,18 @@
     "Context": {
       "ReferenceDateTime": "2019-03-02T00:00:00"
     },
-    "Comment": "PERIODREF. Refine support for quarter, half, week, weekend, etc. Including relatives.",
-    "NotSupported": "dotnet, javascript, python, java",
-    "Results": [
-      {
-        "Text": "primera semana",
-        "Start": 21,
-        "End": 34,
-        "TypeName": "datetimeV2.daterange",
-        "Resolution": {
-          "values": [
-            {
-              "timex": "2019-W01",
-              "type": "daterange",
-              "start": "2018-12-31",
-              "end": "2019-01-07"
-            }
-          ]
-        }
-      }
-    ]
+    "Comment": "Also in English, 'first week' is not extracted ('4th week of month/year' and 'week 3' are the supported patterns).",
```

Comment:
That makes sense, but wouldn't it be better for "semana" to be extracted by itself in "primera semana" as a duration?
Same in English.

Reply:
do you mean to extract a unit like "week" as duration in case it is preceded by an ordinal number?

Reply:
Yes. We can create an issue for it. It's lower priorirty than some of the other fixes.

Reply:
Do you see any potential problem with that extraction?

Reply:
ok, I will open the issue. No I do not see any problem at the moment.